lambda/step-lambda.js

```diff
@@ -17,29 +17,46 @@ const documentClient = new AWS.DynamoDB.DocumentClient({'region': region})
 // result is the returned proomised object (awaits until async function completed) - i promise rejected an exception is thrown
 // NB 'context' parameter has info in it tthat we could find useful and extract, although in this example we don't need it
 
-exports.handler = async function(event, context) {
+exports.handler = function(event, context) {
   //console.log(process.env.AWS_ACCESS_KEY_ID, process.env.AWS_SECRET_ACCESS_KEY, process.env.AWS_SESSION_TOKEN)
   console.log("Received event", JSON.stringify(event))
 
-  const body = JSON.parse(event.body) // converts a JSON string to a JSON object
-  console.log("body:", JSON.stringify(body)) // when logging log the string version otherwise will just say Object
+  const method = event.httpMethod
+  console.log("method:", method)
 
-  const request = {
-    TableName: tablename,
-    Key:{
-      'Id': body.id
-    },
-    UpdateExpression: "ADD SeenCount :step",
-    ExpressionAttributeValues:{
-      ':step': body.step || 1,
-    },
-    ReturnValues:"UPDATED_NEW"
+  switch(method) {
+    case "POST":
+      handlePostRequest(event)
```

Comment:
Currently, this is executing the function and discarding the result.

You'll need a return statement here. Be aware that this function returns a `Promise` so you probably also want an `await`. 
```suggestion
      return await handlePostRequest(event)
```